assets/wizards/site-design/views/main/utils.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+const processFontOptions = ( headingsOnly, options ) =>
+	options.reduce( ( acc, option ) => {
+		const isHeadingsOnly = option.label.indexOf( '(*)' ) > 0;
+		const label = option.label.replace( ' (*)', '' );
+		const selectOption = {
+			label,
+			value: label,
+		};
+		if ( isHeadingsOnly ? headingsOnly : true ) {
+			acc.push( selectOption );
+		}
+		return acc;
+	}, [] );
+
+export const getFontsList = headingsOnly =>
```

Comment:
(Not a blocker) Future enhancement idea? It would be great to source this list from the [Google Fonts API](https://developers.google.com/fonts/docs/developer_api).